src/App.js

```diff
@@ -1,23 +1,42 @@
-import logo from './logo.svg';
+
+import React,  { useEffect } from 'react'
 import './App.css';
+import { useDispatch, useSelector } from 'react-redux'
+import { loadDrinkData } from './redux/actions/liquorSearch.js'
+import { loadGameData } from './redux/actions/boardGame'
+import PairingForm from './Components/PairingForm';
+
+const App = () => {
+  const dispatch = useDispatch()
+  
+  const games = useSelector(state => state.boardGame.allGamesData.games)
+  
+  const getDrinkData = () => {
+    return fetch('https://www.thecocktaildb.com/api/json/v1/1/filter.php?c=Cocktail')
+      .then(res => res.json())
+  }
+
+  const getGameData = () => {
+    return fetch('https://api.boardgameatlas.com/api/search?name=Catan&pretty=true&client_id=JLBr5npPhV')
+      .then(res => res.json())
+  }
+    
+  const getData = () => {
```

Comment:
This was great practice on using Promise.all, but since our APIs are different than we initially thought, let's work on segregating the API calls into another file and taking this back to just the board game fetch on page load.